storage/remote/write.go

```diff
@@ -28,15 +27,15 @@ func (s *Storage) Appender() (storage.Appender, error) {
 func (s *Storage) Add(l labels.Labels, t int64, v float64) (uint64, error) {
 	s.mtx.RLock()
 	defer s.mtx.RUnlock()
-	for _, q := range s.queues {
-		if err := q.Append(&model.Sample{
-			Metric:    labelsToMetric(l),
-			Timestamp: model.Time(t),
-			Value:     model.SampleValue(v),
-		}); err != nil {
-			panic(err) // QueueManager.Append() should always return nil as per doc string.
-		}
-	}
+	// for _, q := range s.queues {
+	// 	if err := q.Append(&model.Sample{
+	// 		Metric:    labelsToMetric(l),
+	// 		Timestamp: model.Time(t),
+	// 		Value:     model.SampleValue(v),
+	// 	}); err != nil {
+	// 		panic(err) // QueueManager.Append() should always return nil as per doc string.
+	// 	}
+	// }
 	return 0, nil
 }
```

Comment:
Drop all this code now.

Reply:
`Add` and `AddFast` could just be no-ops now.